frontend/src/metabase/ui/components/inputs/TextInput/TextInput.styled.tsx

```diff
@@ -1,22 +1,10 @@
 import type { MantineThemeOverride } from "@mantine/core";
-import { getSize } from "@mantine/core";
 
 export const getTextInputOverrides =
   (): MantineThemeOverride["components"] => ({
     TextInput: {
       defaultProps: {
         size: "md",
       },
-      styles: (theme, _, { size = "md" }) => ({
-        wrapper: {
-          "&:not(:only-child)": {
-            marginTop: theme.spacing.xs,
-          },
-        },
-        label: {
-          color: theme.fn.themeColor("text-medium"),
-          fontSize: getSize({ size, sizes: theme.fontSizes }),
-        },
-      }),
```

Comment:
Are the color and font-size on the label no longer needed?

Reply:
They now live on the [label override in the Input component](https://github.com/metabase/metabase/pull/41728/files#diff-41e9d6039ea1166e97f4f8ad12230f21d252bbf686c9ba372f1e072b27d049c8R40-R41) instead.